storybook/toggle-storybook.tsx

```diff
@@ -14,7 +14,7 @@ import { loadString, saveString } from "../app/utils/storage"
  */
 export function ToggleStorybook(props) {
   const [showStorybook, setShowStorybook] = useState(false)
-  const [StorybookUIRoot, setStorybookUIRoot] = useState(null)
+  const [StorybookUIRoot, setStorybookUIRoot] = useState<any>(null)
```

Comment:
I couldn't figure out the correct type for a generic React component. `React.Component` didn't work:

<img width="1257" alt="Screenshot 2021-08-23 at 20 48 04" src="https://user-images.githubusercontent.com/96054/130501220-b0311342-abff-418e-9a0b-8a225fa6c655.png">


Since that's a storybook file and therefore not part of the production app I didn't really care to use `any` here before spending more time on it.

Reply:
I actually though about getting rid of storybooks altogether as I don't use them and maybe never will? Maybe add a `TODO` here just for future reference?

Reply:
Added a TODO to fix the `any` and maybe get rid of storybook in 8077dcb.